lib/help.js

```diff
@@ -30,6 +30,10 @@ Common Options:
   -l | --launch   Specifies the URL to launch during serve.
                     Possible values are host (default), local, or none.
                     Ex: skyux serve --launch local
+  -t | --template Specifies the base template to use when creating a SKY UX application.
```

Comment:
I think your description is great. I think maybe we can flesh it out a little in the docs, but for the help that appears in the command prompt, I think this looks great.